src/styles/index.scss

```diff
@@ -1,3 +1,156 @@
+@mixin margins-content {
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+@mixin font-settings {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
```

Comment:
Consider using this style for the entire body tag to get elements inherited from it